angelsrefining/prototypes/angels-refining-triggers.lua

```diff
@@ -0,0 +1,6 @@
+-- set triggers for the other angels mods
+-- set defaults to start with (may change)
+  --BASIC TRIGGERS
+if mods["angelssmelting"] then
+  angelsmods.trigger.washing_tech = true
```

Comment:
This should move to smelting data.lua to just enable it, instead of putting it in a new file inside refining.

Reply:
Resolved in https://github.com/Arch666Angel/mods/pull/89/commits/efdd289df39a662c5676ec8ef8fef515d91ee952